content/geoip/geolocate-an-ip/web-services.mdx

```diff
@@ -1,14 +1,290 @@
 ---
 draft: false
-title: (todo)
+title: Geolocate an IP using MaxMind Web Services
 ---
 
-(todo)
+Geolocating an IP using MaxMind web services consists of configuring a web
```

Comment:
For the sake of completeness, maybe call this `an IP Address` rather than `an IP`?